### DISCUSSION TITLE
Community maintained rule repository.

### DISCUSSION THREAD

**Discussion by kisst** [evilsocket/opensnitch#1057] [Ideas]
While users who take on the task to control egress, likely to be techie enough to maintain the rules itself too, but it's still a sizable effort and it's done by all the users over and over again, I am thinking to create a repository of rulesets which are following the least privilege principal, and if this is integrated into the UI that would be even better.
So you can look up rules shared by the community which are for that executable which you are currently editing, or it's for the host you are filtering for etc.

This would save users a ton of time, while I am happy to just push up my own rules to a random repo, it would be never be found or used by anyone, the easiest option might be just a simple guideline for repo tag to search for, but many other OSS maintain their own shop so that could be an option too, however the closer it tight to the main project the better chance it has to succeed.

Thoughts ?

**Comment by ItsDrike** [comment]
I'd love to see this come true!

**Comment by Herve5** [comment]
Excellent idea. As for the implementation, I'd say the simplest the better, still targeting lowbrow users like me ;-)
... and wishing you all a happy new year ;-)
